blog/_posts/2019-01-18-fluxdiffeq.md

```diff
@@ -196,42 +194,6 @@ using Plots
 plot(sol)
 ```
 
-![LV Solution Plot](https://user-images.githubusercontent.com/1814174/51388169-9a07f300-1af6-11e9-8c6c-83c41e81d11c.png)
-
-Optional arguments can be passed to control the ODE solver to customise most aspects of the numerical integration.
-For example, if you've some familiarity with the area, a good choice for
-this equation is to use the 5th order Runge-Kutta method `Tsit5()` (related to the `dopri` method,
-or `ode45` in MATLAB). We can also say that we want this solved with a relative error
-tolerance of `1e-8`, and that we want the solution saved at every `0.1` time points. This looks like:
-
-```julia
-sol = solve(prob,Tsit5(),saveat=0.1,reltol=1e-8)
-```
-
-There are a lot more arguments you can pass in to customise most aspects of the ODE solver,
-as

```

Comment:
should we at least link to the docs for discussion about the arguments, and explain the ones we use?